woocommerce/google-listings-and-ads
# Remove Woo Navigation integration

### Changes proposed in this Pull Request:


Remove all WooCommerce Navigation-related  code, hacks and workarounds.


Closes https://github.com/woocommerce/google-listings-and-ads/issues/2406

_Replace this with a good description of your changes & reasoning._


PRs that introduced some navigation-related code:
- https://github.com/woocommerce/google-listings-and-ads/pull/605
- https://github.com/woocommerce/google-listings-and-ads/pull/718
- https://github.com/woocommerce/google-listings-and-ads/pull/743
- https://github.com/woocommerce/google-listings-and-ads/pull/1038
- https://github.com/woocommerce/google-listings-and-ads/pull/1418

As a bonus, I also removed CSS hacks needed for the `useWPBodyMarginOffsetEffect`
removed in https://github.com/woocommerce/google-listings-and-ads/commit/28a80522d40eebef6276b2c840ba24eb6d01f985

### Screenshots:




### Detailed test instructions:


This PR aims to remove eventually dead code and does not change any functionality.

So we need to check for regression if everything works as before with classic navigation.
1. Run automated tests
2. Smoke test everything :goberserk: 
3. Check the main Tab navigation
4. Check Reports sub-navigation
5. Check the layout for full-pages (setup Ads & MC) and full-content ( create/edit campaign, edit address & phone number) pages
6. Check `/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc` look with different window widths (watch out for the top gap)
7. Check if the `Marketing > Google for WooCommerce` is always highlighted for all G4W pages
8. Search the codebase looking for any code related to support WooCommerce Navigation.


### Additional details:


### Changelog entry

> Break - Remove WooCommerce Navigation integration.
